src/Core/Convert.php

```diff
@@ -214,6 +214,24 @@ public static function symbol2sql($identifier, $separator = '.')
     }
 
     /**
+     * @param string $identifier
+     * @param string $separator
+     * @return string
+     */
+    public static function sql2symbol($identifier, $separator = '.')
+    {
+        $q = DB::get_conn()->getIdentifierQuoteCharacter();
```

Comment:
The usage of `$q` is pretty unreadable. Would it make a difference if it's changed to `$quoteChar`?